spec/features/02_view_listings_spec.rb

```diff
@@ -0,0 +1,12 @@
+feature 'view all listings' do
+    scenario 'user can see the list of all bookmarks' do
+        Listing.create(name: 'test listing three')
+        Listing.create(name: 'test listing four')
+        Listing.create(name: 'test listing five')
+
+        visit '/listings'
+        expect(page).to have_css('li', :text => "test listing three")
+        expect(page).to have_css('li', :text => "test listing four")
+        expect(page).to have_css('li', :text => "test listing five")
```

Comment:
is doing this test 3 times necessary? Maybe just once or twice max

Reply:
It was to have some sort of a list but we can just have one or two, same result. I have changed and pushed again.

Reply:
Just realised this is a feature test, and we have already done this test! please can you delete this file